config/deploy.rb

```diff
@@ -7,7 +7,10 @@
 ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
 
 # Default deploy_to directory is /var/www/my_app
-set :deploy_to, '/home/lyberadmin/argo'
+set :deploy_to, '/opt/app/lyberadmin/argo'
```

Comment:
These do not vary per env, so instead of cluttering up the individual env configs, do it once here.